apps/vx-scan/backend/test/helpers/app_helpers.ts

```diff
@@ -206,28 +252,49 @@ export const ballotImages = {
   ],
 } as const;
 
+// Loading of HMPB templates is slow, so in some tests we want to skip it by
+// removing the templates from the ballot package.
+export function createBallotPackageWithoutTemplates(
```

Comment:
Could put this in `@votingworks/test-utils` since it may come in handy for other frontends (VxCentralScan?) - but also fine with leaving it here if that feels preemptive

Reply:
Yeah, I could see it being useful (along with the usb mocking as well). I'm leaning towards leaving them here for now because I've been finding it kind of confusing to track down helpers in many different packages so I'm averse to doing it unless necessary.